handlers/external/external.go

```diff
@@ -344,13 +356,41 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 	return status, nil
 }
 
+func buildQuickRepliesResponse(quickReplies []string, sendMethod string, contentType string) string {
+	var response, format, separator string
+
+	switch {
+	case (sendMethod == http.MethodPost || sendMethod == http.MethodPut) && contentType == contentJSON:
+		response = "[]"
+		format = "[\"%s\"]"
+		separator = "\",\""
+
+	case (sendMethod == http.MethodPost || sendMethod == http.MethodPut) && contentType == contentXML:
+		response = ""
+		format = "<item>%s</item>"
+		separator = "</item><item>"
+
+	default:
+		response = ""
+		format = "%s"
+		separator = "\\,"
+	}
+
+	if len(quickReplies) > 0 {
+		response = fmt.Sprintf(format, strings.Join(quickReplies, separator))
```

Comment:
I don't think this works in all cases. IE, if a quick reply has `/>` in it then that wrecks the format, same goes for one that contains a `"` in a JSON reply.

The quick reply values need to be escaped appropriately according to the content type.